examples/flutter_gallery/lib/demo/material/bottom_navigation_demo.dart

```diff
@@ -95,31 +95,31 @@ class _BottomNavigationDemoState extends State<BottomNavigationDemo>
     _navigationViews = <NavigationIconView>[
       new NavigationIconView(
         icon: const Icon(Icons.access_alarm),
-        title: const Text('Alarm'),
+        title: const Text('Alarm', overflow: TextOverflow.ellipsis),
```

Comment:
We should make the bottom navigation widget do this by default (by setting it on the DefaultTextStyle around its children), the user shouldn't have to explicitly set this.

Reply:
OK, Done.